src/bentoml/_internal/cloud/deployment.py

```diff
@@ -73,26 +73,25 @@ class DeploymentConfigParameters:
     def verify(
         self,
     ):
-        deploy_by_param = (
-            self.name
-            or self.bento
-            or self.cluster
-            or self.access_authorization
-            or self.scaling_min
-            or self.scaling_max
-            or self.instance_type
-            or self.strategy
-            or self.envs
-            or self.extras
+        deploy_by_param = any(
+            param is not None and not "" and len(param) > 0
```

Comment:
What does `not ""` do here?

Reply:
Wanted to check empty string. An oversight. Made a fix